web_src/js/features/comp/TributeExpander.js

```diff
@@ -0,0 +1,59 @@
+import {matchEmoji, matchMention} from '../../utils/match.js';
```

Comment:
Can we call this file and function `TextExpander` instead of `TributeExpander`? The `tribute.js` library will be gone eventually.

Reply:
At least at the moment it is TributeExpander  , and it directly depends on the `tribute`

Reply:
> it directly depends on the tribute

Are you confusing something? I see no import of `tribute.js` in this file.

Reply:
I feel like this file is named confusingly because upper case letters indicate a "component" to me, but it exports only an lowercase `initTributeExpander` function, so I'd say the file should be lowercase too.

Reply:
I didn't confuse it.

The dependency order is: TributeExpander.js -> matchMention -> match.js -> window.config.tributeValues

Reply:
`window.config.tributeValues` is a inaccurate name as well. I would rename to `availableMentions`.

Reply:
Sure, better names are always good.

Reply:
Does it still block? Or could it be renamed later?

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/24400/commits/d22b77f576f89dacc0d04d257c4cbff96c81e693.